x/superfluid/keeper/integration_test.go

```diff
@@ -470,13 +480,22 @@ func (s *TestSuite) TestNativeSuperfluid() {
 		},
 		validatorRewards)
 	s.Require().NoError(err)
-	s.Require().Equal(1, len(validatorRewards.Rewards.Rewards))
-	// After unstaking we will check that those rewards were properly distributed to the delegator
+	// Validators get stake and uosmo. I think this is an issue with test setup
+	s.Require().Equal(2, len(validatorRewards.Rewards.Rewards))
```

Comment:
Would like to get to the bottom of this prior to merge

Reply:
The reason for this is that what gets distributed is every collected fee. That comes from checking the fee collector's balances here: https://github.com/cosmos/cosmos-sdk/blob/main/x/distribution/keeper/allocation.go#L23

Somewhere along the line, someone is sending stake to the fee collector (haven't figured out where, but I think that's ok). The behaviour of distributing that fee is correct though

Reply:
Double emission in test fixed here https://github.com/osmosis-labs/osmosis/pull/8276/commits/63e48b5612f6c9b8bc542884dd1cb256f66cd106